website/docs/docs/deploy/source-freshness.md

```diff
@@ -12,7 +12,14 @@ dbt Cloud provides a helpful interface around dbt's [source data freshness](/doc
 
 ### Enabling source freshness snapshots
 
-First, make sure to configure your sources to [snapshot freshness information](/docs/build/sources#snapshotting-source-data-freshness).
+`dbt build` does _not_ include source freshness checks when it builds and tests resources in your DAG. As such, here's a common pattern for defining jobs:
+- `dbt build` as the run step
+- check box for generating docs
+- check box for source freshness
```

Comment:
This would also be nice as a link to a section below.

Reply:
good callout for both of these suggestions!

Reply:
i've added a checkbox link using `<a id="checkbox"></a>` but because the section is just below - it doesn't seem to make a difference. it may be best to remove to avoid user confusion, what do you think?